plugins/gpio/GPIOPort.cpp

```diff
@@ -44,11 +44,11 @@ bool GPIOOutputPort::Init() {
 }
 
 string GPIOOutputPort::Description() const {
-  vector<uint8_t> pins = m_driver->PinList();
+  vector<uint16_t> pins = m_driver->PinList();
   std::ostringstream str;
   str << "Pins ";
 
-  vector<uint8_t>::const_iterator iter = pins.begin();
+  vector<uint16_t>::const_iterator iter = pins.begin();
```

Comment:
I know this wasn't your code, but can you replace this whole block with our StringJoin function, rather than duplicating code please.